keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/annealing/ResourceActuator.kt

```diff
@@ -0,0 +1,84 @@
+package com.netflix.spinnaker.keel.annealing
+
+import com.netflix.spinnaker.keel.api.Resource
+import com.netflix.spinnaker.keel.persistence.ResourceRepository
+import com.netflix.spinnaker.keel.plugin.ResourceConflict
+import com.netflix.spinnaker.keel.plugin.ResourceHandler
+import com.netflix.spinnaker.keel.plugin.supporting
+import de.danielbechler.diff.ObjectDifferBuilder
+import de.danielbechler.diff.node.DiffNode
+import org.slf4j.LoggerFactory
+import org.springframework.context.event.EventListener
+import org.springframework.scheduling.annotation.Async
+import org.springframework.stereotype.Component
+
+@Component
+class ResourceActuator(
```

Comment:
This is the diff/actuate logic that used to be in `ResourceStateMonitor` but now it just handles one resource at a time via an event.